packages/frontend/src/pages/routes/UnauthenticatedRoutes.tsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import { Routes, Route, Navigate } from 'react-router';
-
+import ManageAccountPage from '../public/ManageAccountPage';
+import HomePage from '../public/HomePage';
 import LandingPage from '../public/LandingPage';
 import SignInPage from '../public/SignInPage';
+import CreateHousePage from '../public/CreateHousePage';
 
 interface UnauthenticatedRoutesProps {}
 
 const UnauthenticatedRoutes: React.FC<UnauthenticatedRoutesProps> = () => {
   return (
     <Routes>
       <Route path="auth" element={<SignInPage />} />
+      <Route path="home" element={<HomePage />} />
+      <Route path="account" element={<ManageAccountPage />} />
+      <Route path="create" element={<CreateHousePage />} />
+      <Route path="*" element={<Navigate to="/home" replace />} />
       <Route path="landing" element={<LandingPage />} />
       <Route path="*" element={<Navigate to="/landing" replace />} />
```

Comment:
These routes are for users than aren't signed in, i.e.
```tsx
path="auth" element = {<SignInPage />} />
path="*" element = {<LandingPage/>} />
```

All other pages should be placed in the `AuthenticatedRoutes` component since only signed in users should be able to access them.